vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/split_test_frequency_cap_unittest.cc

```diff
@@ -51,7 +50,21 @@ TEST_F(BatAdsSplitTestFrequencyCapTest, AllowIfNoGroupInCatalog) {
   EXPECT_FALSE(should_exclude);
 }
 
-TEST_F(BatAdsSplitTestFrequencyCapTest, AllowIfNoFieldTrial) {
+TEST_F(BatAdsSplitTestFrequencyCapTest, DoNotAllowIfNoFieldTrialAndAdGroup) {
+  // Arrange
+  CreativeAdInfo ad;
+  ad.creative_set_id = kCreativeSetId;
+  ad.split_test_group = "GroupA";
+
+  // Act
+  SplitTestFrequencyCap frequency_cap;
+  const bool should_exclude = frequency_cap.ShouldExclude(ad);
+
+  // Assert
+  EXPECT_TRUE(should_exclude);
+}
+
+TEST_F(BatAdsSplitTestFrequencyCapTest, AllowIfFieldTrialAndNoAdGroup) {
```

Comment:
Test logic different to test name. Should add

```
  scoped_refptr<base::FieldTrial> trial = CreateFieldTrial(kTrial);
  trial->AppendGroup(kGroup, 100);
```
in line 72 and remove line 71

Reply:
Fixed